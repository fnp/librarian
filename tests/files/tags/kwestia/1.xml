<naglowek_osoba>PUSTELNIK</naglowek_osoba>

<kwestia>
<strofa>
Więc jako dawniej czynili mocarze,/
Z Lechem się mieniał Scyta na obrączki;/
A pokochawszy mocniej sercem, w darze/
Dał mu koronę... stąd nasza korona./
Zbawiciel niegdyś wyciągając rączki/
Szedł do niej z matki zadumanej łona/
I ku rubinom podawał się cały/
Jako różyczka z liści wychylona,/
I wołał: caca! i na brylant biały/
Różanych ustek perełkami świecił.
</strofa>
</kwestia>
