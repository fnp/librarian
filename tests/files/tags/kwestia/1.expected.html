<div id="book-text">
<h4 id="i1">PUSTELNIK</h4>
<div class="kwestia"><div class="stanza" id="i2">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">Więc jako dawniej czynili mocarze,</div>
<div class="wl verse" id="f2">Z Lechem się mieniał Scyta na obrączki;</div>
<div class="wl verse" id="f3">A pokochawszy mocniej sercem, w darze</div>
<div class="wl verse" id="f4">Dał mu koronę... stąd nasza korona.</div>
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">Zbawiciel niegdyś wyciągając rączki</div>
<div class="wl verse" id="f6">Szedł do niej z matki zadumanej łona</div>
<div class="wl verse" id="f7">I ku rubinom podawał się cały</div>
<div class="wl verse" id="f8">Jako różyczka z liści wychylona,</div>
<div class="wl verse" id="f9">I wołał: caca! i na brylant biały</div>
<a href="#f10" class="wl-num">10</a><div class="wl verse" id="f10">Różanych ustek perełkami świecił.</div>
</div></div>
</div>
